data/human/hails.txt

```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
 
+phrase "friendly civilian"
+	word
+		"Be careful of"
+		"A warning about"
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" "
+	word
+		"those space casinos. I blew"
+	word
+		" "
+	word
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" "
+	word
+		"million credits on "
```

Comment:
If you already have a space after "on," you might as well combine these two words:
```suggestion
	word
		" million credits on "
```